package.json

```diff
@@ -26,9 +26,10 @@
   },
   "homepage": "https://github.com/dancormier/react-native-swipeout",
   "dependencies": {
-    "react-tween-state": "^0.1.3"
+    "react-tween-state": "^0.1.3",
+    "react": "^15.0.0"
   },
   "peerDependencies": {
-    "react-native": "^0.x"
+    "react-native": "^0.27"
```

Comment:
@karanjthakkar simply because it's what i tested with. I know there's another fix PR out as well that might have better requirements.